app/javascript/flavours/glitch/features/notifications/index.js

```diff
@@ -224,8 +226,33 @@ class Notifications extends React.PureComponent {
     this.props.onMarkAsRead();
   };
 
+  groupUpNotifications(notifications, types) {
+    const groupedNotifications = [];
+    for (const notif of notifications) {
+      const newNotif = notif.set('account', ImmutableList([notif.get('account')]));
```

Comment:
This throws a `TypeError: can't access property "set", notif is null`.

Reply:
Strange - I can't reproduce this. Is there a case where notifications contains a null entry?

Reply:
I'm not sure.
It started after I clicked "x others" and happens on every reload now.
I briefly see the "We're sorry, but something went wrong with the Mastodon app" screen and then loads.
Might be a dev vs prod thing though.